pagefaultgames/pokerogue
# [BUG] Fixes status immunity flyout and message not appearing

## What are the changes?

Fixes status immunity flyout and message not appearing.

## Why am I doing these changes?



Simple bug fix.

## What did change?


Status immunity ability should flyout and a message should display when triggered.

### Screenshots/Videos


Before

https://github.com/pagefaultgames/pokerogue/assets/62030095/f2357465-21cd-4790-b4bc-06263a9a8d77

After

https://github.com/pagefaultgames/pokerogue/assets/62030095/fc74f9dc-cf05-40b0-8b98-049fae5d9811

## How to test the changes?




Use src/overrides to give opponent an Effect Immunity ability, give your pokemon status effect moves, see message appear. 

## Checklist
